feat: start the list of conferences (#342)
BedrockStreaming/tech.bedrockstreaming.com: pages/meetup-conference.md

feature-img: "images/travel.jpg"
---

> Bedrock through its actions sponsors events in France, hosts Meetup and conferences in its auditorium.  On many occasions, Bedrock members have even had the opportunity to give conferences in France or abroad.

Reviewer: ```suggestion
Bedrock through its actions, sponsors events in France, hosts Meetup and conferences in its auditorium. Each of those events is an opportunity for Bedrock members to give conferences in France or abroad.
```

However, I feel like something is missing to introduce the content below

feature-img: "images/travel.jpg"
---

Bedrock through its actions, sponsors events in France, hosts Meetup and conferences in its auditorium. Each of those events is an opportunity for Bedrock members to give conferences in France or abroad.

_⭐ sponsored by Bedrock_
_🏠 hosted by Bedrock_

### 2022
- 08/06: ⭐️ 🏠 [LyonJS Meetup](https://www.meetup.com/fr-FR/lyonjs/events/285497869/) 
  - Migration progressive vers Redux Toolkit [Maxime Blanc](https://maximeblanc.fr)
  - Comment ne pas jeter son application Frontend tous les deux ans ? [Florent Dubost](https://twitter.com/fooragnak), [Antoine Caron](https://twitter.com/Slashgear_) 
- 15/05: ⭐️ 🏠 [Docker meetup](https://www.meetup.com/fr-FR/docker-lyon/events/285057478/)

>   - Open Policy Agent avec KICS de chez Checkmarx Jean-Yves Camier

Reviewer 2: ```suggestion
  - Open Policy Agent avec KICS de chez Checkmarx [Jean-Yves Camier](https://twitter.com/Saaraahka)
```
IDK if we want his twitter account to be linked
